opendbc/can/parser.py

```diff
@@ -214,6 +214,9 @@ def can_valid(self) -> bool:
     return self.can_invalid_cnt < CAN_INVALID_CNT and counters_valid
 
   def update(self, strings, sendcan: bool = False):
+    if not strings:
+      return set()
+
```

Comment:
Why this change? Is it necessary?

Reply:
and this is a separate, unrelated change to this PR? you can open a new PR for it